### PR TITLE
added 'handleHover' as buffer for fetching images

### PR DESCRIPTION
**Pull request by James-Jin-Sun**

### CODE REVIEW

File: xartist-frontend/src/app/explore/page.tsx [hunk 1]
```diff
@@ -63,6 +65,17 @@ const Explore = () => {
     }
   };
 
+  // *** Debounced hover handler ***
+  const handleHover = ({ x, y }) => {
+    console.log(x, y);
+    if (hoverTimeoutRef.current) {
+      clearTimeout(hoverTimeoutRef.current);
+    }
+    hoverTimeoutRef.current = setTimeout(() => {
```
Comment (cgfalcon) [inline xartist-frontend/src/app/explore/page.tsx]: The current implementation seems aiming to delay the converting of coordinators. However, as we've discussed yesterday, the problem that we are facing is the performance gap between the producer (hover event in our case) and consumer (flask server in our case). To address this problem, we need introduce a extra storage array to store all the coordinates generated by the hover event, then we need a scheduled timer function to pull the coordinate one by one from the array and convert it into image by calling the server.

Comment (cgfalcon) [reply xartist-frontend/src/app/explore/page.tsx]: In other words, the handleHover function just need to store the coordinates into a array.

**Review by cgfalcon — COMMENTED** [review]
Please see the comments bellow

File: xartist-frontend/src/app/explore/page.tsx (hunk 1 above)
Comment (James-Jin-Sun) [reply xartist-frontend/src/app/explore/page.tsx]: Thank you

**Pull request merged**